src/components/MapSearchBar/MapSearchBar.css

```diff
@@ -0,0 +1,81 @@
+.map-search-panel {
+  position: absolute;
+  right: 0;
+  margin: 20px;
+  z-index: 1000;
+}
+
+.search-icon {
+  height: 25px;
+  position: absolute;
+  padding: 7px 10px;
+  margin: 10px;
+}
+
+.close.icon.clear-search {
+  position: absolute;
+  right: 0;
+  margin: 8px;
+  cursor: pointer;
+}
+
+.close.icon.clear-search:hover {
+  color: #aaa;
+}
+
+.react-autosuggest__input--focused {
+  outline: none;
+}
+
+.react-autosuggest__input {
+  width: 330px;
+  height: 35px;
+  padding: 10px 30px;
+  padding-right: 25px;
+  font-family: Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  border: 1px solid #aaa;
+  border-radius: 4px;
+  z-index: 1000;
+}
+
+.react-autosuggest__input--open {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.react-autosuggest__suggestions-container {
+  display: none;
+}
+
+.react-autosuggest__suggestions-container--open {
+  display: block;
+  position: absolute;
+  top: 51px;
+  width: 330px;
+  border: 1px solid #aaa;
+  background-color: #fff;
+  font-family: Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+  z-index: 2;
+}
+
+.react-autosuggest__suggestions-list {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.react-autosuggest__suggestion {
+  cursor: pointer;
+  padding: 10px 20px;
+  border-bottom: solid 1px #aaa;
```

Comment:
Let's use similar border colors from elements in the right panel. Also, this would cause the very last item in the list to also have a bottom border, which wouldn't be quite what we want. You can use a pseudo-class to override the last element to have no border:

```css
.react-autosuggest__suggestion:last-of-type {
  border-bottom: 0;
}
```